functions-methods.js

```diff
@@ -33,4 +50,16 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+const emailCheckResult = checkEmailValidity("tessmellink@novi.nl");
+
+function checkEmailValidity(email) {
+    if (email.includes("@") && !email.includes(",") && email.charAt(email.length -1) !== ".") {
+        return "true";
+    } else {
+        return "false";
+    }
```

Comment:
Dit zou je nog wat kunnen samenvoegen
```suggestion
    return email.includes("@") && !email.includes(",") && email.charAt(email.length -1) !== ".";
```